tests/network/probes.go

```diff
@@ -80,17 +80,19 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 
 				By("Specifying a VMI with a readiness probe")
 				vmi = createReadyCirrosVMIWithReadinessProbe(virtClient, readinessProbe)
+
+				assertPodNotReady(virtClient, vmi)
```

Comment:
> - IPv6 path
>   The assersion is checked after the VMI is created as before
>   it just moved closer to the VMI creation.

What really matters here is that no logic has been changed and the race has not been resolved for IPv6.
Anything else is just distracting.